phpunit.xml.dist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit backupGlobals="false"
+         backupStaticAttributes="false"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnFailure="true"
+         bootstrap="vendor/autoload.php">
+
+    <testsuite name="SSH Tunnel Test Suite">
```

Comment:
🙈

Reply:
Jo a testy chybí :( IMHO minimálně by se dal napsat jednoduchý na to logování CSV - aby se to nerozbilo, když se updatne keboola/csv